test/membrane_timescaledb_reporter.exs

```diff
@@ -0,0 +1,4 @@
+defmodule Membrane.Telemetry.TimescaleDB do
+  use ExUnit.Case
+  doctest Membrane.Telemetry.TimescaleDB
```

Comment:
Those are not very extensive tests :stuck_out_tongue: It would be great to make sure this package works